apps/accounts/views.py

```diff
@@ -16,15 +15,17 @@
 from rest_framework_expiring_authtoken.authentication import (
     ExpiringTokenAuthentication,
 )
-
+from rest_framework_simplejwt.authentication import JWTAuthentication
+from rest_framework_simplejwt.tokens import RefreshToken
+from .models import JwtToken
 from .permissions import HasVerifiedEmail
 
 from .throttles import ResendEmailThrottle
 
 
 @api_view(["POST"])
 @permission_classes((permissions.IsAuthenticated,))
-@authentication_classes((ExpiringTokenAuthentication,))
+@authentication_classes((JWTAuthentication, ExpiringTokenAuthentication))
```

Comment:
Should we remove the `ExpiringTokenAuthentication` class from here now? We will anyway do it in future. What do you think ?

Reply:
We can't. I mentioned in our call that the frontend will use `ExpiringTokenAuthentication` because we are not planning to override the login view. Otherwise we have to do a db lookup on login to get refresh tokens for login too which we decided not to do.

Reply:
Yes, I remember now. Got it.